internal/provider/namespace_search_attribute_resource.go

```diff
@@ -92,9 +92,10 @@ func (r *namespaceSearchAttributeResource) Schema(ctx context.Context, _ resourc
 				Required:    true,
 			},
 			"type": schema.StringAttribute{
-				CustomType:  internaltypes.CaseInsensitiveStringType{},
-				Description: "The type of the search attribute. Must be one of `bool`, `datetime`, `double`, `int`, `keyword`, or `text`. (case-insensitive)",
-				Required:    true,
+				CustomType:    internaltypes.CaseInsensitiveStringType{},
+				Description:   "The type of the search attribute. Must be one of `bool`, `datetime`, `double`, `int`, `keyword`, or `text`. (case-insensitive)",
```

Comment:
Is this missing the keyword list option?

Reply:
Yes, thanks!
We added support for keyword_list relatively recently.
Pushed an update.